query/tests/it/catalogs/database_catalog.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use chrono::Utc;
+use common_base::tokio;
+use common_datavalues::DataField;
+use common_datavalues::DataSchema;
+use common_datavalues::DataType;
+use common_exception::Result;
+use common_meta_types::CreateDatabaseReq;
+use common_meta_types::CreateTableReq;
+use common_meta_types::DatabaseMeta;
+use common_meta_types::DropDatabaseReq;
+use common_meta_types::DropTableReq;
+use common_meta_types::TableMeta;
+use databend_query::catalogs::Catalog;
+
+use crate::tests::create_catalog;
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+async fn test_catalogs_get_database() -> Result<()> {
+    let catalog = create_catalog().unwrap();
```

Comment:
It would be better change 'unwrap' -> '?' to catch the result instead of panic :)

Reply:
Morning, updated